content/src/repository/extensions/DeploymentsRepository.ts

```diff
@@ -232,6 +232,19 @@ export class DeploymentsRepository {
       return transaction.batch(updates)
     })
   }
+
+  async getDeploymentByHash(contentHash: string): Promise<{ entityId: string }> {
+    return this.db.one(
+      `SELECT ` +
+        `deployment.entity_id ` +
```

Comment:
Do we only need to render the id and not the entire deployment?

Reply:
Yes, since the idea of the endpoint is to validate if there is an active deployment and not the content of the deployment itself.

In the case the user wants the deployment, the deployments or entities endpoint should be used